docs/source/index.mdx

```diff
@@ -32,3 +43,79 @@ It's easy to adopt Apollo incrementally, meaning you can set it up alongside an
     Learn more
   </Button>
 </div>
+
+## Recommended workouts
+
+<MenuWrapper>
+
+<WorkoutMenuItem
+  icon={HeadLight}
+  keyWord="Discover"
+  otherWords="Apollo"
+>
+
+* [The Apollo GraphQL platform](./intro/platform/)
+* [Why GraphQL?](./intro/benefits/)
+* [Introduction to Apollo Server](https://www.apollographql.com/docs/apollo-server/)
+* [Introduction to Apollo Client](https://www.apollographql.com/docs/react/)
+* [Overview of Apollo Graph Manager](https://www.apollographql.com/docs/graph-manager/)
+
+</WorkoutMenuItem>
+
+<WorkoutMenuItem
+  icon={Diving}
+  keyWord="Dive in"
+  otherWords="to the basics"
+>
+
+* Get started with [Apollo Server](https://www.apollographql.com/docs/apollo-server/getting-started/) and [Apollo Client](https://www.apollographql.com/docs/react/get-started/)
+* [Resolvers](https://www.apollographql.com/docs/apollo-server/data/resolvers/) and [Data sources](https://www.apollographql.com/docs/apollo-server/data/data-sources/) (Apollo Server)
+* [Queries](https://www.apollographql.com/docs/react/data/queries/) and [Mutations](https://www.apollographql.com/docs/react/data/mutations/) (Apollo Client)
```

Comment:
In the child doc here, I think we should also be covering operation naming as a best practice

Reply:
Issue created